log/logging_func.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+#Set default config
+logging.root.setLevel(logging.NOTSET)
+
+# Create a custom logger
+logger = logging.getLogger(__name__)
+
+# Create handler
+c_handler = logging.StreamHandler()  #where c_handler is console handler
+c_handler.setLevel(logging.INFO)
```

Comment:
the log level should be an env variable. I noticed you didn't use an env variable at all. If i was scoring the project if it was production ready, i'll score it 2/10

Reply:
😔 that's sad.

Reply:
I'm triggered

Reply:
resolved